src/odemis/gui/comp/text.py

```diff
@@ -903,7 +944,8 @@ def __init__(self, *args, **kwargs):
         choices = kwargs.pop('choices', None)
 
         kwargs['validator'] = IntegerValidator(min_val, max_val, choices)
-        kwargs['key_step'] = kwargs.get('key_step', 1)
+        kwargs['key_step'] = kwargs.get('key_step', 1)  # TODO also replace with fallback?
```

Comment:
Yes, I think it should also use `key_step_min` by default, mostly for the sake of all the classes to behave the same by default.

Reply:
But should it be the calculated version based on max/min or just replace the string as it is?

Reply:
I think you can do exactly the same as the other class, which displays units.